src/audio/drc/tune/sof_drc_build_blob.m

```diff
@@ -1,4 +1,4 @@
-function blob8 = drc_build_blob(blob_struct, endian, ipc_ver)
```

Comment:
Commit title should be **Move** DRC setup scripts to module directory.

Reply:
again i don't understand the benefit of adding "sof_" prefix when we are removing these from a global scope

Reply:
Files are in transition to global scope, hence the sof_ prefix but have been moved to become part of the module source (since the tight coupling between the tools and module). This also allows easier deployment as modules will eventually all become self contained.